cli/cmd/stat.go

```diff
@@ -321,6 +338,11 @@ func buildStatSummaryRequest(resource []string, options *statOptions) (*pb.StatS
 		return nil, err
 	}
 
+	err = options.validate(target.Type)
```

Comment:
Thanks for pulling this out into a separate method. Can you add some tests for it in stat_test.go?